### PR TITLE
Monitors Getting Started

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
- Add monitors getting started
- Clean up monitors section on main Datadog getting started
- Small menu updates

### Motivation
- OKR

### Preview link
https://docs-staging.datadoghq.com/ruth/monitors3/getting_started/application/monitors/

### Additional Notes
N/A

### CODE REVIEW

File: content/en/getting_started/application/_index.md [hunk 1]
```diff
@@ -74,24 +74,23 @@ Dashboards contain [graphs][14] with real-time performance metrics.
...
+## Monitors
 
-{{< img src="getting_started/monitor.png" alt="monitor" responsive="true" >}}
+[Monitors][18] provide alerts and notifications based on metric thresholds, integration availability, network endpoints, and more.
 
-[Monitoring][18] gives you the ability to be notified if the aggregate of a specific metric is above or below a certain threshold—across your entire infrastructure, or per machine (average, max, min, or sum).
+* Use any metric reporting to Datadog
+* Set up multi-alerts (by device, host, etc.)
+* Alert notification messages have `@` capabilities
+* Schedule downtimes for system shutdowns, off-line maintenance, etc.
```
Comment (unclebconnor) [inline content/en/getting_started/application/_index.md]: `Schedule downtimes to suppress notifications for system shutdowns, off-line maintenance, etc.`

File: content/en/getting_started/application/_index.md [hunk 2]
```diff
@@ -74,24 +74,23 @@ Dashboards contain [graphs][14] with real-time performance metrics.
...
-## Monitoring
+## Monitors
 
-{{< img src="getting_started/monitor.png" alt="monitor" responsive="true" >}}
+[Monitors][18] provide alerts and notifications based on metric thresholds, integration availability, network endpoints, and more.
 
-[Monitoring][18] gives you the ability to be notified if the aggregate of a specific metric is above or below a certain threshold—across your entire infrastructure, or per machine (average, max, min, or sum).
+* Use any metric reporting to Datadog
+* Set up multi-alerts (by device, host, etc.)
+* Alert notification messages have `@` capabilities
```
Comment (unclebconnor) [inline content/en/getting_started/application/_index.md]: Use `@` in alert messages to direct notifications to the right people

File: content/en/getting_started/application/monitors.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+Before getting started, you need a [Datadog account][2] linked to a host with the [Datadog Agent][3] installed. To verify, check your [Infrastructure List][4] in Datadog.
+
+## Setup
+
+To create a [metric monitor][5] in Datadog, use the main navigation: *Monitors --> New Monitor --> Metric*.
+
+### Choose the detection method
+
+When you create a metric monitor, **Threshold Alert** is automatically selected as the detection method. A threshold alert compares metric values to a static threshold. The goal for this monitor is to alert on a static threshold, so no change is necessary.
```
Comment (unclebconnor) [inline content/en/getting_started/application/monitors.md]: `alert compares metric values against user defined thresholds` (plural since there are multiple threshold options)

File: content/en/getting_started/application/monitors.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+For this example, the other settings in this section are left on the defaults. For more details, see the [Metric Monitors][7] documentation.
+
+### Say what's happening
+
+Before a monitor can be saved, it must have a title and message.
+
+#### Title
+
+The title must be unique for each monitor. Since this is a multi alert monitor, names are available for each group (`host` and `device`) with message template variables:
```
Comment (unclebconnor) [inline content/en/getting_started/application/monitors.md]: Technically the groups for this monitor are "`host`, `device`" so I'd rephrase to something like:
`names are available for each group element (`host` and `device`)`

File: content/en/getting_started/application/_index.md [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
+title: Getting Started in Datadog
 kind: documentation
 further_reading:
 - link: "https://learn.datadoghq.com/course/view.php?id=2"
   tag: "Learning Center"
   text: "Introduction to Datadog"
 ---
 
-This page gives a high level overview of Datadog's capabilities.
+This page gives a high level overview of the capabilities for the Datadog [US site][1] / [EU site][2].
```
Comment (apigirl) [inline content/en/getting_started/application/_index.md]:
```suggestion
This page gives a high level overview of the capabilities for the Datadog [US site][1] and [EU site][2].
```
